src/MessageList/MessageList.vala

```diff
@@ -295,6 +338,7 @@ public class Mail.MessageList : Gtk.Box {
 
     private void can_move_thread (bool enabled) {
         unowned var main_window = (Gtk.ApplicationWindow) get_toplevel ();
+        move_button.sensitive = enabled;
```

Comment:
Probably should still mark the action here instead of touching button sensitivity directly in case we have other places in the UI where we have a move interface in the future or move action sensitivity is changed in another class etc

Reply:
Got it. To still control the sensitivity I added `ACTON_MARK` (and renamed it to `ACTION_MODIFY`) to the button itself as it is used by the mark button for the same purpose.